### PR TITLE
STK-5365 Structural Clean Up Code - Utils.js

### PR DESCRIPTION
**Pull request by AliArehart**
[Link to Jira ticket](https://strongmind.atlassian.net/browse/STK-5365)

## Purpose 
To resolve spacing issue encountered while working previous ticket in a separate PR, as to not bloat or obfuscate the intent of original work.
Make code cleaner and more readable.

## Approach 
Use IDE tool in Webstorm (command + option + l)

## Testing
Manual testing

## Screenshots/Video
Should behave no differently than before
image

### CODE REVIEW

File: src/CommonCartridge.js [hunk 1]
```diff
@@ -337,7 +337,7 @@ export default class CommonCartridge extends Component {
         result.resourceIdsByHrefMap
       );
     }
-
+    console.log("result", result);
```
Comment (sm-jqualls) [inline src/CommonCartridge.js:340]: Remove console logs

File: src/utils.js [hunk 2]
```diff
@@ -127,6 +127,7 @@ export function parseXml(xml) {
 }
 
 export function parseManifestDocument(manifest, { moduleMeta }) {
+  console.log("Manifest: ", manifest);
```
Comment (sm-jqualls) [inline src/utils.js:130]: Remove console logs

File: src/utils.js [hunk 3]
```diff
@@ -137,6 +138,10 @@ export function parseManifestDocument(manifest, { moduleMeta }) {
   const resources = Array.from(
     manifest.querySelectorAll("resources > resource")
   );
+  console.log(
```
Comment (sm-jqualls) [inline src/utils.js:141]: Remove console logs

**Review by sm-jqualls — APPROVED** [review]

File: src/CommonCartridge.js (hunk 1 above)
Comment (maxamus123) [reply src/CommonCartridge.js:340]: ditto

**Review by maxamus123 — APPROVED** [review]
Once logs are removed I think we are good to go

**Review by alexandraStrongmind — APPROVED** [review]

File: src/CommonCartridge.js (hunk 1 above)
Comment (AliArehart) [reply src/CommonCartridge.js:340]: done.

File: src/utils.js (hunk 2 above)
Comment (AliArehart) [reply src/utils.js:130]: done

File: src/utils.js (hunk 3 above)
Comment (AliArehart) [reply src/utils.js:141]: done.

**Pull request merged**